src/installers.ts

```diff
@@ -643,19 +682,103 @@ const byPriority = (a: Installer, b: Installer) => a.priority - b.priority;
 //
 // Using Vortex parameter names here for convenience.
 //
-export const installerPipeline: Installer[] = [
+export const installerPipeline: InstallerWithPriority[] = [
+  {
+    type: InstallerType.CET,
+    id: "cp2077-cet-mod",
+    testSupported: testForCetMod,
+    install: installCetMod,
+  },
+  /*
+  {
+    type: InstallerType.Redscript,
+    id: "cp2077-redscript-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Red4Ext,
+    id: "cp2077-red4ext-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.TweakDB,
+    id: "cp2077-tweakdb-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.AXL,
+    id: "cp2077-axl-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.INI,
+    id: "cp2077-ini-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Config,
+    id: "cp2077-config-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Reshade,
+    id: "cp2077-reshade-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.LUT,
+    id: "cp2077-lut-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreCET,
+    id: "cp2077-core-cet-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreRedscript,
+    id: "cp2077-core-redscript-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreRed4ext,
+    id: "cp2077-core-red4ext-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreCSVMerge,
+    id: "cp2077-core-csvmerge-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  */
   {
     type: InstallerType.ArchiveOnly,
     id: "cp2077-basic-archive-mod",
-    priority: 30,
-    testSupported: modWithArchiveOnly,
-    install: archiveOnlyInstaller,
+    testSupported: testForArchiveOnlyMod,
```

Comment:
The fallback installer might should be a straight copy from source to destination. This would be the case only if the game being modded is the game the mod is for (based on the given information) and no higher priority installers fit. I think this would be an ideal fallback for most unsupported mods.

Reply:
I’ll add a follow-up PR for this.